lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -414,7 +414,7 @@ def test_query_counts(self, store_type, mongo_count):
 
         RequestCache.clear_request_cache()
 
-        expected_query_count = 43
+        expected_query_count = 44
```

Comment:
@diegomillan Where does this extra query come from?

Reply:
@tomaszgy It's the waffle switch fetch, [see above](https://github.com/edx/edx-platform/pull/18366/files#r200234395).  Needs to be fixed.